crates/extensions/az_mapper_ext/src/lib.rs

```diff
@@ -1,3 +1,66 @@
-pub mod converter;
-pub mod error;
-pub mod size_threshold;
+use tedge_api::mqtt_topics::MqttSchema;
+use tedge_config::models::timestamp::TimeFormat;
+use tedge_config::models::TopicPrefix;
+use tedge_mqtt_ext::Topic;
+
+pub struct AzureConverter {
+    input_topics: String,
+    output_topic: Topic,
+    errors_topic: Topic,
+    add_timestamp: bool,
+    time_format: TimeFormat,
+    size_threshold: usize,
+}
+
+impl AzureConverter {
+    pub fn new(
+        add_timestamp: bool,
+        mqtt_schema: MqttSchema,
+        time_format: TimeFormat,
+        topic_prefix: &TopicPrefix,
+        max_payload_size: u32,
+        input_topics: String,
+    ) -> Self {
+        let output_topic = Topic::new_unchecked(&format!("{topic_prefix}/messages/events/"));
+        let errors_topic =  mqtt_schema.error_topic();
+        let size_threshold = max_payload_size as usize;
+        AzureConverter {
+            input_topics,
+            output_topic,
+            errors_topic,
+            add_timestamp,
+            time_format,
+            size_threshold,
+        }
+    }
+
+    pub fn builtin_flow(&self) -> String {
+        let timestamp_step = if self.add_timestamp {
+            format!(
+                r#"{{ builtin = "add-timestamp", config = {{ property = "time", format = "{time_format}" }} }},"#,
```

Comment:
I'm assuming that this `config` is not just a temporary arrangement while `tedge config` is not exposed to tha flows, but can be used to override the values from `tedge config` even when we expose the same to these builtin flows/transformers.